list/style.css

```diff
@@ -0,0 +1,53 @@
+@import url("../global.css");
+
+body {
+  padding: 2rem;
+}
```

Comment:
Avoid direct tag styling in component style sheets.

Reply:
removed